### DISCUSSION TITLE
Don't find API documentation for this call

### DISCUSSION THREAD

**Discussion by uy5cu71** [community/community#38253] [Apps, API and Webhooks] [answered]
https://api.github.com/users/USERNAME/repos
This will retrieve user public repos and documented in API docs.

But what is this call? 
https://api.github.com/users/USERNAME/subscriptions

Not found any docs about this, what subscriptions?

**Comment by LangLangBart** [comment, marked as answer]
[List repositories watched by user](https://docs.github.com/en/rest/activity/watching#list-repositories-watched-by-a-user)
